.github/workflows/workflow.yml

```
@@ -0,0 +1,77 @@
+name: Test and Deploy
+on:
```

Use the condensed syntax here
```suggestion
on: [pull_request, push]
```